app/util/cleanup_results_dir.py

```diff
@@ -0,0 +1,24 @@
+from pathlib import Path
+import os
+
+ENV_TAURUS_ARTIFACT_DIR = 'TAURUS_ARTIFACTS_DIR'
+FILES_TO_REMOVE = ['jmeter.out',
+                   'jmeter-bzt.properties',
+                   'merged.json',
+                   'merged.yml',
+                   'PyTestExecutor.ldjson',
+                   'system.properties']
+
+if ENV_TAURUS_ARTIFACT_DIR in os.environ:
+    artifacts_dir = os.environ.get('TAURUS_ARTIFACTS_DIR')
```

Comment:
Pls use here variable ENV_TAURUS_ARTIFACT_DIR

Reply:
It is ridiculous, what is wrong with me? ))) I need to be more collected